apps/web/src/pages/templates/workflow/RegularDigestMetadata.tsx

```diff
@@ -27,12 +26,11 @@ export const RegularDigestMetadata = ({ index, readonly }: { index: number; read
           control={control}
           name={amountFieldName}
           defaultValue={amountDefaultValue}
-          render={({ field, fieldState }) => {
+          render={({ field }) => {
             return (
               <Input
                 {...field}
                 value={field.value || ''}
-                error={showErrors && fieldState.error?.message}
```

Comment:
The error message expands the input field width, so removed that. There won't be any validation error cases with this field, because the default value is set on blur.